src/FactoryTrait.php

```diff
@@ -191,14 +191,13 @@ public function factory($seed, $defaults = [], string $prefix = null): object
      * add prefix.
      *
      * Rules observed, in order:
-     *  - If class starts with "." then prefixing is always done.
-     *  - If class contains "\" prefixing is never done.
+     *  - If class starts with "./" then prefixing is always done.
+     *  - If class contains "\" or it is an anonymous class prefixing is never done.
      *  - If class (with prefix) exists, do prefix.
      *  - don't prefix otherwise.
      *
      * Example: normalizeClassName('User', 'Model') == 'Model\User';
      * Example: normalizeClassName(Test\User::class, 'Model') == 'Test\User'; # or as per "use"
-     * Example: normalizeClassName('Test/User', 'Model') == 'Model\Test\User';
      * Example: normalizeClassName('./User', 'Model') == 'Model\User';
      * Example: normalizeClassName('User', 'Model') == 'Model\User'; # if exists, 'User' otherwise
```

Comment:
Comment # if exists, User otherwise is wrong here. I do not see any class_exists() method call here.

Reply:
?

Reply:
That comment at the end of this line. We do not check for class existence in this method, just normalize class name.

Reply:
You mean `or as per "use"`? I left this untouched, I basically removed `\` support only.